### ISSUE TITLE
The One Habit That Stops Procrastination In Its Tracks

### ISSUE THREAD

**Issue by szwnba** [szwnba/affpops#42]
![](https://www.asianefficiency.com/wp-content/uploads/2017/04/ctn-header.jpg)

So what are you waiting for?

It is a good question to ask yourself about your goals, priorities, and most important tasks. Have you ever reached the end of the day and asked yourself, “What did I really get done today?” You may have everything you need to accomplish your goals, yet you still fight to find your focus. One of the causes for this is procrastination. It is one of the most common productivity traps. Sometimes this happens even if you have been busy. Some days it is much easier to do things that just use your time instead of finishing your most important tasks. In this article we are going to tell you how to stop procrastinating.

What is procrastination?
------------------------

Procrastination is when you delay or postpone something. It is the invisible force between what you intend to do and taking action on it. When you know what you need to get done, but you lack the focus to make it happen. It is easier to plan something than to actually do it.

### Procrastination is the Friction Between You and Your Goals

The most effective way to stop procrastinating is to remove friction between you and your most important tasks.

Friction is procrastination’s best friend. Think of friction as a hurdle that stops you from doing something immediately. Let’s say that you want to add drinking of water immediately after waking up as part of your morning ritual. But every time you wake up, you end up checking your phone first because it’s right beside your bed and you still need to get up and go to the kitchen just to get water. The friction point in that example is that the water is not readily available.

Sounds simple enough, right? But if you think about all the time you procrastinated on your goals and the panic that ensues to make sure you finish some tasks, it stops being a simple issue. When you procrastinate daily, your todo list also stacks up. We all do not like a never-ending to-do list but when you procrastinate, it’s inevitable.

Just think of the last time you procrastinated. Why did you put it off? Where was the friction point? Chances are it falls in one of these three buckets:

### The Three Buckets of Procrastination

![](https://www.asianefficiency.com/wp-content/uploads/2018/04/buckets-500x334.jpg)

1.  Getting to clarity about the outcome you’re hoping to achieve
2.  Getting started on a task (even though you know exactly what to do)
3.  Lack of energy and willpower to do anything (especially if it seems challenging)

After working with so many people on their procrastination challenges, [we’ve identified that most procrastination challenges](https://www.asianefficiency.com/podcast/167-procrastination/) fall into one of three buckets. And in my experience, the first one is the most common pitfall. At Asian Efficiency, we’ve helped more than 15 thousand of our customers and readers get clarity on what they want to achieve. Having clarity helped them stop procrastinating and actually get stuff done. Let’s face it. You should not be sacrificing what’s important to you because you end up procrastinating all the time end up not get anything done.

When you don’t have clarity about the outcome, it’s easy to put things off until the last minute. To put it in other words: if you are motivated about achieving a goal and you know all the benefits of achieving that goal, you’re more likely to get started.

Clarity For Long Term Goals
---------------------------

Most people who procrastinate never get to clarity. We address this extensively in our paid online courses ([especially in the Dojo, the biggest productivity training library on the market](https://shop.asianefficiency.com/dojo/)), but today we will cover all the other ways people procrastinate, and what you can do about it. Even when you have clarity, we still procrastinate for various reasons.

No matter how motivated I am to grow Asian Efficiency and how acutely aware I am of all the benefits that come with that, there are days when I wake up and don’t feel like putting in the work.

We all have these kinds of days. What do you then?

**Eliminate every friction point you have in your life.**

A lot of procrastination techniques we teach in [our paid courses](https://shop.asianefficiency.com/dojo/) address friction points. When you remove common friction points it becomes easy to get started on things you need to do.

In fact, we teach all of our clients one specific habit that will eliminate most of the friction points in your life. We call it **Clearing To Neutral**.

**Sidenote:** _When other productivity experts join the AE team, this is one of the first things I teach them. Not only is it one of the core concepts AE is known for, but I also want them to be able to implement it themselves and get all the benefits from it. As we like to say, happy people are productive people._

What is Clearing To Neutral?
----------------------------

The idea is very simple: how can you make your life easy for “future you” to get started?

In restaurants, the process of cleaning the grill is very important. It ensures the grill will last longer, the food will taste better, and you prevent any bacteria from growing. Before the restaurant closes, the people in the kitchen always clean the grill so the next day when they come in it is ready for use.

Imagine for a moment that you’re the chef. You come in, highly motivated to cook delicious dishes for your guests. As soon as you step into the kitchen, you see pots and pans laying on the floor. There are dirty dishes in the sink. The grill is covered in grease and the place smells like rotten eggs.

Would you still be motivated to cook for your guests?

At this point, you lost all your [desire to do anything](https://www.asianefficiency.com/motivation/eliminating-motivation-doing-the-right-thing/). Before you can get to your specialty (cooking), you have to mop the floors, wash the dishes, inspect food twice to make sure it’s not expired, and clean the grill. By the time you’re done, you have no more energy left to cook and get in your zone of genius.

That’s how sometimes our lives are:

*   Before we can write our book, we’re so disorganized with our notes, papers and digital files that we get frustrated and in a bad mood by the time we sit down to write.
*   When you walk into your office, you have to clear your desk, put stuff in different places, shred papers, and only then are you ready to open your computer to start work.
*   When you open your laptop you see your browser with 34 tabs open, a Youtube video that’s resuming play, and 12 notifications coming up on the sidebar. Now you’re triggered and distracted, and your plan to sit down to do your weekly review is now in jeopardy.

Can you see how easy it is to procrastinate?

When you have so much friction before you get to do the actual thing you wanted to do in the first place, it’s no wonder people procrastinate.

### Clearing to Neutral Will Help You Stop Procrastinating

With Clearing to Neutral, a habit we introduced to the world back in 2012 that changed the game, you eliminate the initial friction.

The reason we call it CTN is because whenever you finish an activity, you need to clear everything so it is in a neutral position. When something is neutral, it’s ready to be used right away.

Finish brushing your teeth? Put the toothbrush in its place so it’s ready for next time.

As you wrap up your writing, close all browser tabs and apps on your computer.

When you finish cooking, clean the grill, pots, and pans so they are ready for use next time.

This is why the habit of Clearing To Neutral is so important: **it prevents you from procrastinating in the future**. By making sure you set everything up for “future you,” [you make it easy for “future you” to hit the ground running](https://www.asianefficiency.com/mindsets/inevitability-mindset/). There’s no friction to getting started (which is why most of us procrastinate).

### 3 Phases of Taking Action

_For my fellow systems thinkers, here’s another way to think about it: everything has 3 phases._

1.  _A setup phase_
2.  _An execution phase_
3.  _A reset phase_

_As we always talk about at AE, everything in life is a system. Most of us have the initial two phases figured out but miss the third one. That’s really what Clearing To Neutral introduces._

We recommend you introduce a Clearing To Neutral habit on top of every habit you have. This is an easy place to start.

### 3 Every Day Examples for Clearing to Neutral

*   After you’re done with your morning ritual, Clear To Neutral by setting everything up for tomorrow.
*   Before you leave the office, Clear To Neutral your desk, computer and any inboxes you have.
*   Anytime you’re about to get away from your computer, Clear To Neutral by closing all apps and browser tabs.

Virtually any habit you have will benefit from having a CTN action added at the end. Just think about your future self. How can you make it easy for future YOU to stop procrastinating?

Taking Action
-------------

Here’s how we recommend you start:

1.  Write down 3 rituals that first come to mind.
2.  Ask yourself for each ritual: “what can I do at the end of this ritual to make it easy for my “future self” to implement the ritual?”
3.  Add this to your newly updated ritual.

The last thing I’d like you to do is to close all your browser tabs and clear your desk before you call it a day. I know, it sounds so simple, but please do it today. Your future self will thank me tomorrow.
